MovieTime/MovieTime/View/Movies/MTMovieDetailViewController.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  MTMovieDetailViewController.swift
+//  MovieTime
+//
+//  Created by Enrique Melgarejo on 11/01/18.
+//  Copyright © 2018 Choynowski. All rights reserved.
+//
+
+import UIKit
+
+class MTMovieDetailViewController: MTViewController {
+
+    // MARK: Outlets
+    @IBOutlet weak var imageViewBackdrop: UIImageView!
+    @IBOutlet weak var activityIndicatorBackdrop: UIActivityIndicatorView!
+    @IBOutlet weak var imageViewPoster: UIImageView!
+
+    @IBOutlet weak var labelMovieName: UILabel!
+    @IBOutlet weak var labelMovieGenres: UILabel!
+    @IBOutlet weak var labelScore: UILabel!
+    @IBOutlet weak var labelLanguage: UILabel!
+
+    @IBOutlet weak var labelMovieDuration: UILabel!
+    @IBOutlet weak var labelReleaseDate: UILabel!
+
+    @IBOutlet weak var textViewOverview: UITextView!
```

Comment:
I forgot to unmark the `Behaviour: Editable` on the Storyboard. It can be improved by keeping the `Behavior: Selectable` so users can only select (copy) the information.